package.json

```diff
@@ -20,6 +20,9 @@
   },
   "author": "Matthew Sumner, Justin Kenyon and contributors",
   "license": "MIT",
+  "dependencies": {
+    "torii": "~0.2.3"
```

Comment:
are you okay with the loosened dep? (this allows semver bugfixes but not significant feature additions)